lib/easy_meli.rb

```diff
@@ -16,4 +16,21 @@ def self.configuration
   def self.configure
     yield(configuration)
   end
+
+  def self.authorization_url(country_code, redirect_uri)
+    EasyMeli::AuthorizationClient.authorization_url(country_code, redirect_uri)
+  end
+
+  def self.create_token(code, redirect_uri, logger: nil)
+    EasyMeli::AuthorizationClient.create_token(code, redirect_uri, logger: nil)
```

Comment:
why you are passing `nil` to logger param if you have defined the logger in the method declaration?

Reply:
Ups

Reply:
Listo